### PR TITLE
Install script: fix query latest release from GitHub on missing curl

### PR DESCRIPTION
**Pull request by perazz**
Improve the command to query the latest release version from Github by trying curl then wget; with fallback to the last known version release if none of the two commands is available

### CODE REVIEW

**Comment by perazz** [comment]
@fortran-lang/admins please assign any reviewers if you wish to close this PR, thanks.

File: install.sh [hunk 1]
```diff
@@ -21,10 +21,21 @@ usage()
...
+       curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+       grep '"tag_name":'        |                                       # Get tag line
+       sed -E 's/.*"([^"]+)".*/\1/' |                                    # Pluck JSON value
+       sed -E 's/^v//'                                                   # Remove heading "v" if present
+    elif command -v wget > /dev/null 2>&1; then
+       wget -q -O- "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+       grep '"tag_name":'        |                                       # Get tag line
+       sed -E 's/.*"([^"]+)".*/\1/' |                                    # Pluck JSON value
+       sed -E 's/^v//'                                                   # Remove heading "v" if present
+    else
```
Comment (awvwgk) [inline install.sh:35]: You can reuse the logic later in the script where we define the `FETCH` variable.

Comment (perazz) [reply install.sh]: Looks better now - made it a function

**Review by awvwgk — APPROVED** [review]
Looks good to me now.

**Pull request merged**